### ISSUE TITLE
[BUG] New-PnPTeamsTeam does not create a team

### ISSUE THREAD

**Issue by Geo-Ron** [pnp/powershell#794]
### Reporting an Issue or Missing Feature
The command New-PnPTeamsTeam does not create a team and shows NO output whatsoever.
It does create a M365 group, the SP site is created, the exchange properties are configured.

Using Get-PnPMicrosoft365Group returns 'HasTeam = $false'

### Expected behavior 
I would expect a Teams enabled M365 group to be created

### Actual behavior 
No output of the command (even with use of -Verbose)
M365 group is created, but without Team

### Steps to reproduce behavior
```powershell
New-PnPTeamsTeam -DisplayName "myPnPDemo1" -Visibility Private -AllowCreateUpdateRemoveTabs $false -AllowUserDeleteMessages $false -Owner $owner[0]
Get-PnPMicrosoft365Group mypnpdemo1
```

### What is the version of the Cmdlet module you are running?
pnp.powershell version 1.6.0

### Which operating system/environment are you running PnP PowerShell on?
- [x] Windows
- [ ] Linux
- [ ] MacOS
- [ ] Azure Cloud Shell
- [ ] Azure Functions
- [ ] Other : please specify

### Current workaround

```powershell
Connect-PnPOnline using certificate authentication setup using https://pnp.github.io/powershell/articles/authentication.html#setting-up-access-to-your-own-azure-ad-app
$Group = New-PnPMicrosoft365Group -DisplayName myPnPDemo1 -Isprivate:$True -CreateTeam -Owners $owners -Members $owners -MailnickName "myPnPDemo1-msTeams" -Description "myPnPDemo1-Description"
$Team = Get-PnPTeamsTeam -Identity $Group.MailNickName
Set-PnPTeamsTeam -Identity $Team.GroupId -AllowUserDeleteMessages:$false -AllowCreateUpdateRemoveTabs:$False

```

Possibly linked to #613

**Comment by Geo-Ron** [comment]
Workaround not working for me: see #796

**Comment by Geo-Ron** [comment]
It seems like this is fixed in #778 ... awaiting new release...

**Comment by gautamdsheth** [comment]
Has been fixed in the latest nightly, please try that.
